core/server/api/v2/webhooks.js

```diff
@@ -0,0 +1,41 @@
+const models = require('../../models');
+const common = require('../../lib/common');
+
+module.exports = {
+    docName: 'webhooks',
+    add: {
+        statusCode: 201,
+        headers: {},
+        options: [
+            'include'
+        ],
+        permissions: true,
+        query(frame) {
+            return models.Webhook.getByEventAndTarget(
+                frame.data.webhooks[0].event,
+                frame.data.webhooks[0].target_url,
+                frame.options
+            ).then((webhook) => {
+                if (webhook) {
+                    return Promise.reject(
+                        new common.errors.ValidationError({message: common.i18n.t('errors.api.webhooks.webhookAlreadyExists')})
+                    );
+                }
+
+                return models.Webhook.add(frame.data.webhooks[0], frame.options);
+            });
+        }
+    },
+    destroy: {
+        statusCode: 204,
+        headers: {},
+        options: [
+            'include',
+            'id'
```

Comment:
id is required, isn't it? 🤔

Reply:
Yeah it is, also 'include' is not needed I think. Just updated